2017-07-01 10:35:41 ERROR ServiceDaemon:60 - Exception encountered during startup.
com.vng.csm.configuration.ConfigException: Invalid configuration file.
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:23)
	at com.wahsis.iot.common.Config.init(Config.java:25)
	at com.wahsis.iot.main.ServiceDaemon.main(ServiceDaemon.java:32)
Caused by: org.apache.commons.configuration.ConfigurationException: Cannot locate configuration source wahsis_cloud.conf
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:217)
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:197)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.load(AbstractHierarchicalFileConfiguration.java:164)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.<init>(AbstractHierarchicalFileConfiguration.java:91)
	at org.apache.commons.configuration.HierarchicalINIConfiguration.<init>(HierarchicalINIConfiguration.java:232)
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:21)
	... 2 more
2017-07-01 10:35:41 ERROR ServiceDaemon:62 - Uncaught exception: Invalid configuration file.
com.vng.csm.configuration.ConfigException: Invalid configuration file.
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:23)
	at com.wahsis.iot.common.Config.init(Config.java:25)
	at com.wahsis.iot.main.ServiceDaemon.main(ServiceDaemon.java:32)
Caused by: org.apache.commons.configuration.ConfigurationException: Cannot locate configuration source wahsis_cloud.conf
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:217)
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:197)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.load(AbstractHierarchicalFileConfiguration.java:164)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.<init>(AbstractHierarchicalFileConfiguration.java:91)
	at org.apache.commons.configuration.HierarchicalINIConfiguration.<init>(HierarchicalINIConfiguration.java:232)
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:21)
	... 2 more
2017-07-01 10:36:31 ERROR ServiceDaemon:60 - Exception encountered during startup.
com.vng.csm.configuration.ConfigException: Invalid configuration file.
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:23)
	at com.wahsis.iot.common.Config.init(Config.java:25)
	at com.wahsis.iot.main.ServiceDaemon.main(ServiceDaemon.java:32)
Caused by: org.apache.commons.configuration.ConfigurationException: Cannot locate configuration source wahsis_iot_cloud.conf
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:217)
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:197)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.load(AbstractHierarchicalFileConfiguration.java:164)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.<init>(AbstractHierarchicalFileConfiguration.java:91)
	at org.apache.commons.configuration.HierarchicalINIConfiguration.<init>(HierarchicalINIConfiguration.java:232)
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:21)
	... 2 more
2017-07-01 10:36:31 ERROR ServiceDaemon:62 - Uncaught exception: Invalid configuration file.
com.vng.csm.configuration.ConfigException: Invalid configuration file.
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:23)
	at com.wahsis.iot.common.Config.init(Config.java:25)
	at com.wahsis.iot.main.ServiceDaemon.main(ServiceDaemon.java:32)
Caused by: org.apache.commons.configuration.ConfigurationException: Cannot locate configuration source wahsis_iot_cloud.conf
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:217)
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:197)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.load(AbstractHierarchicalFileConfiguration.java:164)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.<init>(AbstractHierarchicalFileConfiguration.java:91)
	at org.apache.commons.configuration.HierarchicalINIConfiguration.<init>(HierarchicalINIConfiguration.java:232)
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:21)
	... 2 more
2017-07-01 10:37:39 ERROR ServiceDaemon:60 - Exception encountered during startup.
com.vng.csm.configuration.ConfigException: Invalid configuration file.
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:23)
	at com.wahsis.iot.common.Config.init(Config.java:25)
	at com.wahsis.iot.main.ServiceDaemon.main(ServiceDaemon.java:32)
Caused by: org.apache.commons.configuration.ConfigurationException: Cannot locate configuration source wahsis_iot_cloud.conf
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:217)
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:197)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.load(AbstractHierarchicalFileConfiguration.java:164)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.<init>(AbstractHierarchicalFileConfiguration.java:91)
	at org.apache.commons.configuration.HierarchicalINIConfiguration.<init>(HierarchicalINIConfiguration.java:232)
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:21)
	... 2 more
2017-07-01 10:37:39 ERROR ServiceDaemon:62 - Uncaught exception: Invalid configuration file.
com.vng.csm.configuration.ConfigException: Invalid configuration file.
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:23)
	at com.wahsis.iot.common.Config.init(Config.java:25)
	at com.wahsis.iot.main.ServiceDaemon.main(ServiceDaemon.java:32)
Caused by: org.apache.commons.configuration.ConfigurationException: Cannot locate configuration source wahsis_iot_cloud.conf
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:217)
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:197)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.load(AbstractHierarchicalFileConfiguration.java:164)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.<init>(AbstractHierarchicalFileConfiguration.java:91)
	at org.apache.commons.configuration.HierarchicalINIConfiguration.<init>(HierarchicalINIConfiguration.java:232)
	at com.vng.csm.configuration.INIFileConfiguration.<init>(INIFileConfiguration.java:21)
	... 2 more
2017-07-01 10:38:26  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:38:27  INFO local:839 - local: Created a new connection
2017-07-01 10:38:27  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:38:27  INFO local:839 - local: Created a new connection
2017-07-01 10:38:27  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:38:27  INFO local:839 - local: Created a new connection
2017-07-01 10:38:27  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:38:27  INFO local:839 - local: Created a new connection
2017-07-01 10:38:27  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:38:27  INFO local:839 - local: Created a new connection
2017-07-01 10:38:27  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:38:27  INFO local:839 - local: Created a new connection
2017-07-01 10:38:28  INFO MqttManager:57 - mqtt broker = tcp://127.0.0.1:1883
2017-07-01 10:38:28 ERROR MqttManager:68 - MqttManager.start: Unable to connect to server
Unable to connect to server (32103) - java.net.ConnectException: Connection refused (Connection refused)
	at org.eclipse.paho.client.mqttv3.internal.TCPNetworkModule.start(TCPNetworkModule.java:79)
	at org.eclipse.paho.client.mqttv3.internal.ClientComms$ConnectBG.run(ClientComms.java:650)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.eclipse.paho.client.mqttv3.internal.TCPNetworkModule.start(TCPNetworkModule.java:70)
	... 2 more
2017-07-01 10:38:28  INFO log:178 - Logging initialized @2843ms
2017-07-01 10:38:28  INFO WebServer:73 - Start server...
2017-07-01 10:38:28  WARN ContextHandler:1324 - o.e.j.s.h.ContextHandler@38887256{/,null,null} contextPath ends with /*
2017-07-01 10:38:29  INFO Server:301 - jetty-9.1.5.v20140505
2017-07-01 10:38:29  INFO ContextHandler:747 - Started o.e.j.s.h.ContextHandler@39b3529c{/static,null,AVAILABLE}
2017-07-01 10:38:29  INFO ContextHandler:747 - Started o.e.j.s.h.ContextHandler@38887256{/cloud/ntf,null,AVAILABLE}
2017-07-01 10:38:29  INFO ContextHandler:747 - Started o.e.j.s.ServletContextHandler@230a39b4{/,null,AVAILABLE}
2017-07-01 10:38:29  INFO ServerConnector:266 - Started ServerConnector@1a8e3988{HTTP/1.1}{0.0.0.0:9103}
2017-07-01 10:38:29  INFO ServerConnector:266 - Started ServerConnector@7afac462{HTTP/1.1}{0.0.0.0:9104}
2017-07-01 10:38:29  INFO Server:350 - Started @3307ms
2017-07-01 10:41:27  INFO local:839 - local: Destroyed connection
2017-07-01 10:41:27  INFO local:839 - local: Destroyed connection
2017-07-01 10:41:27  INFO local:839 - local: Destroyed connection
2017-07-01 10:41:27  INFO local:839 - local: Destroyed connection
2017-07-01 10:41:27  INFO local:839 - local: Destroyed connection
2017-07-01 10:41:32  INFO local:839 - local: Destroyed connection
2017-07-01 10:45:21  INFO ServiceDaemon:48 - Shutdown thread before webserver getinstance
2017-07-01 10:45:21  INFO ServerConnector:303 - Stopped ServerConnector@1a8e3988{HTTP/1.1}{0.0.0.0:9103}
2017-07-01 10:45:21  INFO ServerConnector:303 - Stopped ServerConnector@7afac462{HTTP/1.1}{0.0.0.0:9104}
2017-07-01 10:45:21  INFO ContextHandler:864 - Stopped o.e.j.s.ServletContextHandler@230a39b4{/,null,UNAVAILABLE}
2017-07-01 10:45:21  INFO ContextHandler:864 - Stopped o.e.j.s.h.ContextHandler@38887256{/cloud/ntf,null,UNAVAILABLE}
2017-07-01 10:45:21  INFO ContextHandler:864 - Stopped o.e.j.s.h.ContextHandler@39b3529c{/static,null,UNAVAILABLE}
2017-07-01 10:45:42  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:45:43  INFO local:839 - local: Created a new connection
2017-07-01 10:45:43  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:45:43  INFO local:839 - local: Created a new connection
2017-07-01 10:45:43  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:45:43  INFO local:839 - local: Created a new connection
2017-07-01 10:45:43  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:45:43  INFO local:839 - local: Created a new connection
2017-07-01 10:45:43  INFO local:839 - local: Getting connection (user/password): jdbc:mysql://127.0.0.1:3306/wahsis?autoReconnect=true&failOverReadOnly=false&maxReconnects=10&useUnicode=true&characterEncoding=UTF-8&noAccessToProcedureBodies=true
2017-07-01 10:45:43  INFO local:839 - local: Created a new connection
2017-07-01 10:45:43  INFO MqttManager:57 - mqtt broker = tcp://127.0.0.1:1883
2017-07-01 10:45:44 ERROR MqttManager:68 - MqttManager.start: Unable to connect to server
Unable to connect to server (32103) - java.net.ConnectException: Connection refused (Connection refused)
	at org.eclipse.paho.client.mqttv3.internal.TCPNetworkModule.start(TCPNetworkModule.java:79)
	at org.eclipse.paho.client.mqttv3.internal.ClientComms$ConnectBG.run(ClientComms.java:650)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.eclipse.paho.client.mqttv3.internal.TCPNetworkModule.start(TCPNetworkModule.java:70)
	... 2 more
2017-07-01 10:45:44  INFO log:178 - Logging initialized @3437ms
2017-07-01 10:45:44  INFO WebServer:73 - Start server...
2017-07-01 10:45:44  WARN ContextHandler:1324 - o.e.j.s.h.ContextHandler@5e30dedf{/,null,null} contextPath ends with /*
2017-07-01 10:45:44  INFO Server:301 - jetty-9.1.5.v20140505
2017-07-01 10:45:45  INFO ContextHandler:747 - Started o.e.j.s.h.ContextHandler@65853f02{/static,null,AVAILABLE}
2017-07-01 10:45:45  INFO ContextHandler:747 - Started o.e.j.s.h.ContextHandler@5e30dedf{/cloud/ntf,null,AVAILABLE}
2017-07-01 10:45:45  INFO ContextHandler:747 - Started o.e.j.s.ServletContextHandler@231afddf{/,null,AVAILABLE}
2017-07-01 10:45:45  INFO ServerConnector:266 - Started ServerConnector@1f4e0f6d{HTTP/1.1}{0.0.0.0:9103}
2017-07-01 10:45:45  INFO ServerConnector:266 - Started ServerConnector@4dd97c40{HTTP/1.1}{0.0.0.0:9104}
2017-07-01 10:45:45  INFO Server:350 - Started @4233ms
